TanukiKit/User.swift

```diff
@@ -28,30 +28,23 @@ import RequestKit
     open var canCreateProject: Bool?
     open var twoFactorEnabled: Bool?
     open var external: Bool?
-    open var privateToken: String?
 
     public init(_ json: [String: Any]) {
         if let id = json["id"] as? Int {
             name = json["name"] as? String
             login = json["username"] as? String
             self.id = id
             state = json["state"] as? String
-            if let urlString = json["avatar_url"] as? String, let url = URL(string: urlString) {
-                avatarURL = url
-            }
-            if let urlString = json["web_url"] as? String, let url = URL(string: urlString) {
-                webURL = url
-            }
+            if let urlString = json["avatar_url"] as? String, let url = URL(string: urlString) { avatarURL = url }
+            if let urlString = json["web_url"] as? String, let url = URL(string: urlString) { webURL = url }
             createdAt = Time.rfc3339Date(string: json["created_at"] as? String)
             isAdmin = json["is_admin"] as? Bool
             bio = json["bio"] as? String
             location = json["location"] as? String
             skype = json["skype"] as? String
             linkedin = json["linkedin"] as? String
             twitter = json["twitter"] as? String
-            if let urlString = json["website_url"] as? String, let url = URL(string: urlString) {
-                websiteURL = url
-            }
+            if let urlString = json["website_url"] as? String, let url = URL(string: urlString) { websiteURL = url }
```

Comment:
I preferred verbose mode, please revert this change as it is hard to read in one line.